src/main/scala/io/iohk/ethereum/network/TestSocketHandshaker.scala

```diff
@@ -64,7 +64,7 @@ object TestSocketHandshaker {
       msgs.collect {
         case m: Status =>
           sendMessage(m, frameCodec, out) //send same status message
-          sendMessage(GetBlockHeaders(Left(5000), maxHeaders = 20, skip = 0, reverse = 0), frameCodec, out) //ask for block further in chain to get some transactions
+          sendMessage(GetBlockHeaders(Left(1000000), maxHeaders = 1, skip = 0, reverse = 0), frameCodec, out) //ask for block further in chain to get some transactions
```

Comment:
Do you want to change this or did it went through accidentally? it's not big deal though, just curious.